app/facets/participant/participant.coffee

```diff
@@ -12,12 +12,21 @@ class Participant
 				id: @id
 				display_name: user.displayName
 
-		@game_session_id = new nx.Cell
 
 		@game_session = new nx.Cell
-			'<-': [@game_session_id, (id) => @service.game_sessions.get id]
 			action: (game_session) =>
-				game_session.add_participant @
+				game_session?.add_participant @
+
+		@game_session_id = new nx.Cell
+			'->': [
+				((id) =>
+					if @service.game_sessions.has id
+						@game_session
+					else
+						[]
```

Comment:
Right.
Additionally, refer to https://github.com/DaQuirm/nexus/wiki/Bindings-and-Cells#demux-binding